crates/pathfinder/Cargo.toml

```diff
@@ -3,7 +3,7 @@ name = "pathfinder"
 version = "0.4.5"
 edition = "2021"
 license = "MIT OR Apache-2.0"
-rust-version = "1.64"
+rust-version = "1.65"
```

Comment:
What caused the msrv bump? Just curious.

Reply:
The Cairo 1.0 compiler requires 1.65:

```
error[E0658]: `let...else` statements are unstable
 --> /home/hidden/.cargo/git/checkouts/cairo-f086c7e6d4098a68/c38fd76/crates/cairo-lang-syntax/src/node/utils.rs:7:5
  |
7 | /     let Some(parent) = node.parent() else {
8 | |         return false;
9 | |     };
  | |______^
  |
  = note: see issue #87335 <https://github.com/rust-lang/rust/issues/87335> for more information
```

Reply:
Ah yeah the new let if else feature. Okay cool.